icees_api/features/sql.py

```diff
@@ -1002,17 +1002,18 @@ def compute_multivariate_table(conn, table_name, year, cohort_id, feature_variab
     if feat_len < 3:
         raise HTTPException(status_code=400, detail="At least three feature variables must be provided "
                                                     "for computing multivariate associations")
-    associations = []
+
     # get feature_constraint list from the first feature variable
     feat_constraint_list = []
     levels0 = get_feature_levels(feature_variables[0], year=year)
     for level in levels0:
         non_op_idx = 0
-        for lev in level:
-            if lev in ['<', '>', '=']:
-                non_op_idx += 1
-            else:
-                break
+        if isinstance(level, str):
```

Comment:
What happens if the level isn't a string?

Reply:
@maximusunc Good question. If the level isn't a string, the extra processing in the if condition will be skipped. The logic in the if condition is for checking values enclosed in a string with operators included, e.g., ```>9``` for InPatientEDVisits feature variable, for multivariate analysis, which separate the operator from the value. If the level isn't a string, it assumes the value can be directly used without any extra check and processing.